atlassian/bitbucket.py

```diff
@@ -10,11 +10,12 @@
 class Bitbucket(AtlassianRestAPI):
     bulk_headers = {"Content-Type": "application/vnd.atl.bitbucket.bulk+json"}
 
-    def __init__(self, *args, **kwargs):
-        super(Bitbucket, self).__init__(*args, **kwargs)
-        url = kwargs.pop('url', False)
-        if url and 'bitbucket.org' in url:
-            self.cloud = True
+    def __init__(self, **kwargs):
```

Comment:
Do we really need to skip the *args ? 
https://www.python.org/dev/peps/pep-0362/#boundarguments-object

Reply:
There are no args, only kwargs.